src/property-filter/index.tsx

```diff
@@ -203,6 +204,21 @@ const PropertyFilter = React.forwardRef(
       parsedText.step === 'property' &&
       getExtendedOperator(filteringProperties, parsedText.property.key, parsedText.operator)?.form;
 
+    const tokenValueToLabel = useMemo(() => {
```

Comment:
Good point! I would expect all textual operators (`=`, `!=`, `:`, `!:`) to work. I'll check if that's the case and add tests. It is not expected to use the `label` property with other operators such as `<`, `>`. However, I wouldn't add a dev-warning or something here because we support custom operator matchers.

Reply:
Actually, the labelled ones require the 1:1 relation so the `:` and `!:` operators are not going to be supported. I can make this explicit in the docs.